### ISSUE TITLE
SA1009 false positive with null conditional operator

### ISSUE THREAD

**Issue by elyoh** [ptittof57/Visual-StyleCop#10]
The following snippet:
```c#
var posts = new List<string>() { "abc", "123" };
posts.First()?.Replace('a', 'z');
```
incorrectly gives:
SA1009 : CSharp.Spacing : Invalid spacing around the closing parenthesis.
for the second line.

Build 4.7.53 and previous build 23/08/2015.

**Comment by ptittof57** [comment]
Fixed for next release
